starboard/loader_app/BUILD.gn

```diff
@@ -342,8 +334,8 @@ static_library("reset_evergreen_update") {
   deps = [ "//starboard:starboard_group" ]
 }
 
-# TODO: b/309493306 - Stop building evergreen targets for all non-evergreen platforms.
-if (!is_host_win) {
+if (!sb_is_modular && sb_is_evergreen_compatible &&
```

Comment:
we want to build this target for modular builds as it's under starboard and raspi only builds modularly + evergreen

Reply:
Overlooked it, fixed